test/integration/render-tests/globe/globe-transition/at-transition-zoom/style.json

```diff
@@ -0,0 +1,59 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "description": "Tests which tiles ought to be rendered near the transition zoom.",
+      "width": 1105,
+      "height": 676,
+      "operations": [
+        ["setProjection", "globe"],
+        ["wait"]
+      ]
+    }
+  },
+  "center": [-107.017, 37.612],
+  "zoom": 5.99,
+  "sources": {
+    "quad": {
+      "type": "geojson",
+      "data": {
+        "type": "Polygon",
+        "coordinates":  [
+          [
+            [
+              -115.041,
+              31.267
+            ],
+            [
+              -100.041,
+              31.267
+            ],
+            [
+              -100.041,
+              42.267
+            ],
+            [
+              -115.041,
+              42.267
+            ],
+            [
+              -115.041,
+              37.267
+            ]
+          ]
+        ]
+      }
+    }
+  },
+  "layers": [
+    {
+      "id": "fill",
+      "type": "fill",
+      "source": "quad",
+      "paint": {
+        "fill-antialias": false,
+        "fill-color": "rgba(0,0,255,0.5)"
+      }
+    }
+  ]
+}
```

Comment:
This render test confirms that the bug exists. Should we instead introduce a fix for the issue and use this render test to validate the fix? I would be fine moving the fix to a follow-up PR too.

Reply:
Yes indeed I think this bug can actually be fixed separately because it will also affect the circle and heatmap layers when globe view is enabled.